### DISCUSSION TITLE
An Interactive Extrapolation Toolkit for HTA in R

### DISCUSSION THREAD

**Discussion by LaureNF357** [HTAhackathon2024/.github#19] [Ideas]
### What is the problem we're trying to solve

Develop a user friendly (Shiny) app that feeds on survival data specifically, and interactively fit and compare different extrapolation models. Implement uncertainty quantification, validation tools, performance metrics, economic evaluation modules, etc

### Which tools will we use?

R

**Comment by n8thangreen** [comment]
I wrote some code for survival extrapolation for some training previously here
https://icon-in-r.github.io/surv-extrapolation/

Perhaps this could be useful to get the ball rolling?

**Reply by n8thangreen** [reply]
It seems there are some already like
`survint`
https://dgallacher.shinyapps.io/survint/
And
`survextrap` (by the same person)
Amongst others.

May need to review these first and identify gaps?

**Reply by alex-vand** [reply]
Haven't played around with it but there's also [easysurv](https://maple-health-group.github.io/easysurv/) which might be worth adding to the review / gap assessment list.

**Reply by CKENNEDY-HE** [reply]
Could be good to add a function which reports if/when the hazard< gen pop mortality hazard. Also a table reporting S(t) at 1,2 5 and 10 years

**Comment by shubhrampandey** [comment]
I am developing something similar, and it's currently in progress. See here:

https://heorlytics.shinyapps.io/survlytics/

**Comment by LaureNF357** [comment]
Interesting to read all the various highlights. Thank you. Getting familiar with all the potential extrapolation methods (for survival and other data type as well), particularly those involving Bayesian techniques, is what inspired me to come up with this topic...
